src/cmd/policy.go

```diff
@@ -13,12 +13,19 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
 type regoInput struct {
 	Files []string `json:"files"`
 }
 
+type GithubRepo struct {
```

Comment:
Forgot to delete this.  I added this when I was trying to set result to a struct.  Don't need to do it now since I converted the response body to io.Reader and did the same thing that the OPA docs had.